examples/structured/credit_card_fraud.exs

```diff
@@ -1,12 +1,14 @@
 Mix.install([
   {:axon, "~> 0.5"},
+  {:polaris, "~> 0.1"},
   {:exla, "~> 0.5"},
   {:nx, "~> 0.5"},
-  {:explorer, "~> 0.5"}
+  {:explorer, path: "../explorer"} # 0.5.8-dev
```

Comment:
This won't work.
Pending update to future explorer release.

Reply:
Updated now to point to latest explorer release.
https://github.com/elixir-nx/axon/pull/508/commits/38135fa45bef6c2d30bb477d4fa2c91ee1f97d88